src/services/legal-services.ts

```diff
@@ -252,58 +253,98 @@ export default class LegalServices {
     })
   }
 
-  /** Business Digital Credential Helpers **/
+  /**
+   * Gets a pre-signed URL for the specified filename.
+   * @param filename the file name
+   * @returns the presigned url object
+   */
+  static async getPresignedUrl (fileName: string): Promise<PresignedUrlIF> {
+    const url = `documents/${fileName}/signatures`
+    return axios.get(url)
+      .then(response => response?.data)
```

Comment:
Note: this returns the (typed) data object from the response. Not all methods in this class do that.